algorithms/schema-free/cosine_batch.gsql

```diff
@@ -0,0 +1,60 @@
+CREATE QUERY cosine_batch(string vertex_type, string edge_type, string edge_attribute, int topK, bool print_accum = true, string file_path, string similarity_edge, int           num_of_batches=1) {
+  TYPEDEF TUPLE<vertex ver, float val> res_tup;
```

Comment:
add comments for the meaning of accumulators and each select statement.

Also, please refer to the coding style of other algorithms.